src/Components/Books.js

```diff
@@ -40,28 +44,73 @@ export default function Books() {
   return (
     <>
       <div className="container">
-        <div className="my-4 border border-3 p-0">
+        <div className="my-4 p-0 add-gap">
           {
           bookArr.map((e) => (
-            <div key={data.id} className="d-flex justify-content-between ms-0 p-3">
-              <span>
-                Title:
-                {' '}
-                {e.title}
-              </span>
-              <span>
-                Author:
-                {' '}
-                {e.author}
-              </span>
-              <button
-                className="btn btn-primary mb-3"
-                type="submit"
-                onClick={() => { handleDelete(e.id); }}
-              >
-                Remove
+            <div key={data.id} className="ms-0 p-3 book">
+              <div className="book-info">
+                <h4 className="book-category">
+                  {' '}
+                  {e.category}
+                </h4>
+                <h2 className="book-title">
+                  {' '}
+                  {e.title}
+                </h2>
+                <h6 className="book-author">
+                  {' '}
+                  {e.author}
+                </h6>
+                <div className="action-buttons">
+                  <button
+                    className="button-outline"
+                    type="submit"
+                  >
+                    Comments
 
-              </button>
+                  </button>
+                  <div className="vertical-divider" />
+                  <button
+                    className="button-outline"
+                    type="submit"
+                    onClick={() => { handleDelete(e.id); }}
+                  >
+                    Remove
+
+                  </button>
+                  <div className="vertical-divider" />
+                  <button
+                    className="button-outline"
+                    type="submit"
+                  >
+                    Edit
+
+                  </button>
+                </div>
+              </div>
+              <div className="">
+                <div style={{ width: 80, height: 80 }}>
```

Comment:
- [ ] Inline styling is bad CSS practice, kindly remove it and add it on your css file